### ISSUE TITLE
Uncompatibility With Enhanced Visuals Mod

### ISSUE THREAD

**Issue by kenijey** [tterrag1098/Blur#6]
![1](https://cloud.githubusercontent.com/assets/24831406/26679892/b44ad580-46df-11e7-8e3a-704fb6328610.png)
![2](https://cloud.githubusercontent.com/assets/24831406/26679893/b4fd5dcc-46df-11e7-9268-7c183cc1563c.png)
enhanced visuals adds some screen effects, and they get disabled if i move my mouse into an item on JEI.
don't know what causes, but i think i had better to mention @creativemd :D

**Comment by kenijey** [comment]
using forge 2310, EnhancedVisuals v1.1.2 mc1.11.2,  Blur-1.0.1-9

**Comment by kenijey** [comment]
![123](https://cloud.githubusercontent.com/assets/24831406/26681793/fefeb428-46e6-11e7-9249-827b8e69074d.png)
and ah, it's a gui by other mod (levelup!)

**Comment by tterrag1098** [comment]
So this is not a Blur issue?

**Comment by CreativeMD** [comment]
Nope, i doubt that this is an issue of Blur, looks like this is just a EnhancedVisuals issue. Created a new issue on my repository so we can try to solve it there.

**Comment by CreativeMD** [comment]
@tterrag1098 any updates about the IFacade hook for Chisel?
